### PR TITLE
Fix `header-stats` command

### PR DESCRIPTION
**Pull request by hansieodendaal**
## Description
Tari-script network reset omitted changing this `stibbons` reference.

## Motivation and Context
See above

## How Has This Been Tested?
Ran the `header-stats` command in the base node

## Checklist:

* [X] I'm merging against the `tari-script` branch.
* [X] I have squashed my commits into a single commit.

### CODE REVIEW

**Review by sdbondi — APPROVED** [review]
Really need the global config to be passed into the command handler - but this is fine for a quick fix

**Comment by hansieodendaal** [comment]
> Really need the global config to be passed into the command handler - but this is fine for a quick fix

Yes, much better. Done

File: applications/tari_base_node/src/command_handler.rs [hunk 1]
```diff
@@ -1013,6 +1014,15 @@ impl CommandHandler {
                 )
                 .unwrap();
 
+                if header.header.hash() != header.accumulated_data.hash {
```
Comment (sdbondi) [inline applications/tari_base_node/src/command_handler.rs:1015]: Nice

**Review by sdbondi — APPROVED** [review]
Looks good!

File: applications/tari_base_node/src/command_handler.rs (hunk 1 above)
Comment (hansieodendaal) [reply applications/tari_base_node/src/command_handler.rs:1015]: At @mikethetike's suggestion

**Review by SWvheerden — APPROVED** [review]
LGTM

**Review by StriderDM — APPROVED** [review]
+1

**Pull request merged**